### DISCUSSION TITLE
Lighthouse positioning system accuracy

### DISCUSSION THREAD

**Discussion by apark2459** [bitcraze/discussions#1490] [Q&A] [answered]
So I understand that, ideally, the crazyflie shouldn't be any closer than 50 cm from the basestation for accurate measurement. What is the maximum distance that the basestation can accurately measure up to? I just want to get a sense of how high up from the ground I need to set up the basestations.

Also, my lab plans to use the system for robots that move around on a water surface. Might this be a problem (with reflections, etc.)?

Thank you!

**Comment by knmcguire** [comment, marked as answer]
So it refers to mostly the mounted height of the lighthouse basestations. If the basestation is mounted at 2 meters height, then we recommend the crazyflie to not fly higher than 1.50 meters. However, in reality you might be able to fly a bit higher anyway, it's just a general rule of thumb.

It might be okay as long as the reflection doesn't reflect back into the image sensor, so reflective areas on the side of the flight area is an issue. water might cause a reflection from the bottom, so I think it will be okay.

**Reply by apark2459** [reply]
Thank you!
